### DISCUSSION TITLE
how to classify stories in storybook v6?

### DISCUSSION THREAD

**Discussion by uyarn** [storybookjs/storybook#11967] [Help]
we are going to migrate storybook from v5 to v6.
in v5, we use symbol '|'  at the title of each stories to classify the components .
such as:
`export default {
  title: 'General | BlankLink',
  parameters: {
    component: BlankLink,
  },
}
`
`export default {
  title: 'DataEntry | Input',
  parameters: {
    component:Input,
  },
}
`
then the storybook would show the classification of components. However, in v6 it seems to be out of work.
Anybody know how to change in v6?

**Comment by shilman** [comment]
Yeah, use `'General/BlankLink'` etc.

https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#removed-hierarchy-separators
